operator/k8s_identity.go

```diff
@@ -103,14 +113,31 @@ func identityGCIteration(ctx context.Context) {
 				log.WithError(err).WithFields(logrus.Fields{
 					logfields.Identity: identity,
 				}).Error("Deleting unused identity")
+
+				if operatorOption.Config.EnableMetrics {
+					failedRuns++
+					metrics.IdentityGCRuns.WithLabelValues(metrics.LabelValueOutcomeFail).Set(float64(failedRuns))
+				}
```

Comment:
a failed run would only likely be incremented whenever `ctx.Err() != nil` before we return from this function.